patches/chrome-android-BUILD.gn.patch

```diff
@@ -8,7 +8,7 @@ index b067a3f7af514bc48632365427ba8ecffa4885df..dadadab8dd95e47bc228053bd1335cbd
    ]
 +  deps += brave_chrome_app_java_resources_deps sources += brave_java_resources resource_overlay = true
  }
- 
```

Comment:
Please, don't remove the space on "empty lines" in patches. Some Windows versions of git will fail to apply the patch if the space is missing.